src/Simplex/Messaging/Crypto.hs

```diff
@@ -243,3 +245,13 @@ rsaPrivateKey pk =
       R.private_dQ = undefined,
       R.private_qinv = undefined
     }
+
```

Comment:
this file probably got into pr by a mistake

Reply:
just didn't want to delete this function - let it be :)